openc3/python/openc3/packets/limits.py

```diff
@@ -159,6 +159,9 @@ def set(
         if green_low and green_high:
             limits_for_set[4] = float(green_low)
             limits_for_set[5] = float(green_high)
+        else:
+            limits_for_set[4] = None
+            limits_for_set[5] = None
```

Comment:
Ruby calls `delete_at` if the existing limits has green low and green ... I think it's cleaner to just clear it in the else